src/docdb/registerDocDBCommands.ts

```diff
@@ -67,7 +67,8 @@ export function registerDocDBCommands(editorManager: CosmosEditorManager): void
             node = <DocDBStoredProcedureTreeItem>await ext.tree.showTreeItemPicker([DocDBStoredProcedureTreeItem.contextValue]);
         }
         await editorManager.showDocument(new DocDBStoredProcedureNodeEditor(node), node.label + '-cosmos-stored-procedure.js');
-    });
+        // tslint:disable-next-line:align
+    }, 200);
```

Comment:
How about creating a const?

Reply:
Sounds good. 

Does the 200 value seem agreeable? I don't quite know what value VSCode is comfortable with (200 worked in my test, but I couldn't find any documentation regarding minimum values)

Reply:
I'd use what Nathan used.